templates/mjs/app.js.ejs

```diff
@@ -2,8 +2,8 @@
 import createError from 'http-errors';
 <% } -%>
 import express from 'express';
```

Comment:
Grouping Node.js modules at the top of the file and Express import afterward is good practice, leaving a blank line between the two blocks.

```
import path from 'path';
import { fileURLToPath } from 'url';

import express from 'express';
```